### DISCUSSION TITLE
Access to the CK Editor and Redactor API via add-on settings

### DISCUSSION THREAD

**Discussion by liamcrean** [ExpressionEngine/ExpressionEngine#2881] [Feature Requests]
Seems like pro would be a great time to introduce a little more control over RTE. 

In particular common problems like selecting the types of heading available from the styles menu (rarely do we want to offer H1 as an option to editors - v bad SEO), or greater control over images (flex/grid), for example.

Both editors have API's that allow fairly easy settings control via JSON (I believe). Wygwam offers this - but as a pro user I'd expect the same level of function from on-board RTE fields.

Also having control over styles/fonts/sizing inside the field would be great — e.g. The default font for redactor is at odds with the EE control panel UI and I'd love to be able to introduce some v.basic layout elements from the front-end here

**Comment by TomJaeger** [comment]
Thanks for passing this along. I believe most (if not all) of this is set to be released in 7.2  Specifically in this PR. https://github.com/ExpressionEngine/ExpressionEngine/pull/2339
